src/main/java/com/powsybl/openloadflow/network/LfContingency.java

```diff
@@ -152,6 +152,7 @@ public void apply(LoadFlowParameters.BalanceType balanceType) {
             generator.setDisabled(true);
             if (generator.getGeneratorControlType() != LfGenerator.GeneratorControlType.OFF) {
                 generator.setGeneratorControlType(LfGenerator.GeneratorControlType.OFF);
+                bus.getReactivePowerControl().ifPresent(rc -> rc.updateReactiveKeys());
```

Comment:
@geofjamg I think this is missing for voltage control. @caioluke is going to publish an issue.

Reply:
I just opened a new PR with the fix: https://github.com/powsybl/powsybl-open-loadflow/pull/904